packages/at_client/lib/src/service/notification_service.dart

```diff
@@ -166,6 +166,7 @@ class NotificationParams {
 
   OperationEnum get operation => _operation;
 
+  @Deprecated('Use only MessageType.key, MessageType.text has been deprecated ')
```

Comment:
@Deprecated('Use MessageType.key')

Reply:
addressed in [cef655a](https://github.com/atsign-foundation/at_client_sdk/pull/1314/commits/cef655aa946933208e86c9e356ae1ef6f3f5d965)